src/main/java/gov/nasa/pds/tools/validate/BundleManager.java

```diff
@@ -533,6 +626,38 @@ public static void makeException(URL url, String location) {
 
         ArrayList<Target> ignoreCollectionList = BundleManager.buildCollectionIgnoreList(parentURL,url);
         LOG.debug("post_call:buildCollectionIgnoreList:url {}",url);
+        if (ignoreCollectionList != null) {
+            LOG.debug("makeException:url,ignoreCollectionList {},{},{}",url,ignoreCollectionList.size(),ignoreCollectionList);
+            // Because this function has knowledge of why a particular file is being ignored while crawling
+            // we will log it here.
+            for (Target target : ignoreCollectionList) {
+                    LOG.info("SKIP: {} due to logical_identifier the same as selected collection but different version",target.getUrl());
+                    // Write a record to report that we are skipping this file.
+                    if (BundleManager.m_report != null) {
+                        try {
+                            // In our case, we want ExceptionType to be WARNING so it gets printed in the report for default verbose level of 3 (WARNING)
+                            ValidationProblem p1 = null;
+                            if ((FlagsUtil.getSeverity() != null) &&
+                                (FlagsUtil.getSeverity().isInfoApplicable() || FlagsUtil.getSeverity().isDebugApplicable())) {
```

Comment:
@qchaupds similar to previous comment, I may be wrong, but I don't think we need any of this custom logic to handle this reporting. Let's just use the existing reporting logic based upon the set `ExceptionType`. In this case, the `ExceptionType` should be changed to `ExceptionType.INFO`.

Reply:
@jordanpadams Per your comment, the code has been simplified in the parent and left the logic what to print in the Report.java class.    Please have another look at the last code commit.

Re-run the 3 tests resulted in the following "SKIP" status:

validate -R pds4.bundle --skip-context-reference-check --skip-content-validation -r report_github372_bundle_invalid_without_v.json   -s json -t src/test/resources/github69/invalid/bundle_kaguya_derived.xml >& t10

validate -R pds4.bundle -v 1 --skip-context-reference-check --skip-content-validation -r report_github372_bundle_invalid_with_v_1.json   -s json -t src/test/resources/github69/invalid/bundle_kaguya_derived.xml >& t11

validate -R pds4.bundle -v 0 --skip-context-reference-check --skip-content-validation -r report_github372_bundle_invalid_with_v_0.json   -s json -t src/test/resources/github69/invalid/bundle_kaguya_derived.xml >& t12


grep -A 6 SKIP report_github372_bundle_invalid_with*.json
report_github372_bundle_invalid_without_v.json:      "status": "SKIP",
report_github372_bundle_invalid_without_v.json-      "label": "file:/home/qchau/sandbox/validate/src/test/resources/github69/invalid/bundle_kaguya_derived_2.xml",
report_github372_bundle_invalid_without_v.json-      "messages": [
report_github372_bundle_invalid_without_v.json-        {
report_github372_bundle_invalid_without_v.json-          "severity": "WARNING",
report_github372_bundle_invalid_without_v.json-          "type": "info.integrity.unreferenced_file",
report_github372_bundle_invalid_without_v.json-          "message": "Skipping file:/home/qchau/sandbox/validate/src/test/resources/github69/invalid/bundle_kaguya_derived_2.xml due to not being selected as the bundle target"
--
report_github372_bundle_invalid_without_v.json:      "status": "SKIP",
report_github372_bundle_invalid_without_v.json-      "label": "file:/home/qchau/sandbox/validate/src/test/resources/github69/invalid/data_spectra/spectra_data_collection_inventory_2.xml",
report_github372_bundle_invalid_without_v.json-      "messages": [
report_github372_bundle_invalid_without_v.json-        {
report_github372_bundle_invalid_without_v.json-          "severity": "WARNING",
report_github372_bundle_invalid_without_v.json-          "type": "info.integrity.unreferenced_file",
report_github372_bundle_invalid_without_v.json-          "message": "Skipping file:/home/qchau/sandbox/validate/src/test/resources/github69/invalid/data_spectra/spectra_data_collection_inventory_2.xml due to logical_identifier the same as selected collection but different version"
--
report_github372_bundle_invalid_with_v_0.json:      "status": "SKIP",
report_github372_bundle_invalid_with_v_0.json-      "label": "file:/home/qchau/sandbox/validate/src/test/resources/github69/invalid/bundle_kaguya_derived_2.xml",
report_github372_bundle_invalid_with_v_0.json-      "messages": [
report_github372_bundle_invalid_with_v_0.json-        {
report_github372_bundle_invalid_with_v_0.json-          "severity": "WARNING",
report_github372_bundle_invalid_with_v_0.json-          "type": "info.integrity.unreferenced_file",
report_github372_bundle_invalid_with_v_0.json-          "message": "Skipping file:/home/qchau/sandbox/validate/src/test/resources/github69/invalid/bundle_kaguya_derived_2.xml due to not being selected as the bundle target"
--
report_github372_bundle_invalid_with_v_0.json:      "status": "SKIP",
report_github372_bundle_invalid_with_v_0.json-      "label": "file:/home/qchau/sandbox/validate/src/test/resources/github69/invalid/data_spectra/spectra_data_collection_inventory_2.xml",
report_github372_bundle_invalid_with_v_0.json-      "messages": [
report_github372_bundle_invalid_with_v_0.json-        {
report_github372_bundle_invalid_with_v_0.json-          "severity": "WARNING",
report_github372_bundle_invalid_with_v_0.json-          "type": "info.integrity.unreferenced_file",
report_github372_bundle_invalid_with_v_0.json-          "message": "Skipping file:/home/qchau/sandbox/validate/src/test/resources/github69/invalid/data_spectra/spectra_data_collection_inventory_2.xml due to logical_identifier the same as selected collection but different version"
--
report_github372_bundle_invalid_with_v_1.json:      "status": "SKIP",
report_github372_bundle_invalid_with_v_1.json-      "label": "file:/home/qchau/sandbox/validate/src/test/resources/github69/invalid/bundle_kaguya_derived_2.xml",
report_github372_bundle_invalid_with_v_1.json-      "messages": [
report_github372_bundle_invalid_with_v_1.json-        {
report_github372_bundle_invalid_with_v_1.json-          "severity": "WARNING",
report_github372_bundle_invalid_with_v_1.json-          "type": "info.integrity.unreferenced_file",
report_github372_bundle_invalid_with_v_1.json-          "message": "Skipping file:/home/qchau/sandbox/validate/src/test/resources/github69/invalid/bundle_kaguya_derived_2.xml due to not being selected as the bundle target"
--
report_github372_bundle_invalid_with_v_1.json:      "status": "SKIP",
report_github372_bundle_invalid_with_v_1.json-      "label": "file:/home/qchau/sandbox/validate/src/test/resources/github69/invalid/data_spectra/spectra_data_collection_inventory_2.xml",
report_github372_bundle_invalid_with_v_1.json-      "messages": [
report_github372_bundle_invalid_with_v_1.json-        {
report_github372_bundle_invalid_with_v_1.json-          "severity": "WARNING",
report_github372_bundle_invalid_with_v_1.json-          "type": "info.integrity.unreferenced_file",
report_github372_bundle_invalid_with_v_1.json-          "message": "Skipping file:/home/qchau/sandbox/validate/src/test/resources/github69/invalid/data_spectra/spectra_data_collection_inventory_2.xml due to logical_identifier the same as selected collection but different version"

Reply:
@jordanpadams The ExceptionType used now is INFO.

Reply:
Re-ran the above commands again and the severity is now INFO:

{pds-dev3.jpl.nasa.gov}/home/qchau/sandbox/validate 139 % grep -A 6 SKIP report_github372_bundle_invalid_with*.json | grep severity

report_github372_bundle_invalid_with_v_0.json-          "severity": "INFO",
report_github372_bundle_invalid_with_v_0.json-          "severity": "INFO",
report_github372_bundle_invalid_with_v_1.json-          "severity": "INFO",
report_github372_bundle_invalid_with_v_1.json-          "severity": "INFO",

The only troubling thing is the report file report_github372_bundle_invalid_without_v.json does not have the "SKIP" line.
Do you know why?